src/api.ts

```diff
@@ -105,6 +107,7 @@ class RippleAPI extends EventEmitter {
     super()
     validate.apiOptions(options)
     this._feeCushion = options.feeCushion || 1.2
```

Comment:
Not a bad idea, but I think people are paying a lot more due to `load_factor` increases. I don't have evidence of this, though.

A few drops doesn't make much difference in comparison, and it will ensure that transactions have some priority over "super economy" txs paying just the base fee.